packages/lib-classifier/src/plugins/tasks/dropdown-simple/components/SimpleDropdownTask.js

```diff
@@ -103,26 +89,24 @@ function SimpleDropdownTask({
       <Box
         gap='xsmall'
       >
-        <Select
```

Comment:
I'd leave the Select in, as a drop down menu when the number of options is small. Autosuggest/combobox is only needed when the number of options is large.

Reply:
For clarity, the Select component and the input+datalist component would never coexist - is that correct?

Reply:
As per the conversation thread, the search via datalist is enabled for all SimpleDropdownTasks.